proof-systems/src/darlin/pcd/simple_marlin.rs

```diff
@@ -18,12 +15,58 @@ use crate::darlin::{
     },
 };
 use poly_commit::ipa_pc::Commitment;
+use std::ops::Deref;
 use std::marker::PhantomData;
 
+#[derive(Derivative)]
+#[derivative(Clone(bound = ""), Debug(bound = ""), Eq(bound = ""), PartialEq(bound = ""))]
+#[derive(CanonicalSerialize, CanonicalDeserialize)]
+pub struct MarlinProof<G: AffineCurve, D: Digest>(pub Proof<G::ScalarField, InnerProductArgPC<G, D>>);
```

Comment:
This again points at moving away from being as generic as possible. 
I like it!